src/content/docs/en/concepts/why-astro.mdx

```diff
@@ -1,45 +1,63 @@
 ---
 title: Why Astro?
-description: "Astro is an all-in-one web framework for building fast, content-focused websites. Learn more."
+description: "Astro is an all-in-one web framework for building fast, content-driven websites. Learn more."
 i18nReady: true
 ---
 
-Astro is an **all-in-one** **web framework** for building **fast,** **content-focused websites.**
+**Astro** is a web framework for building **content-driven websites** that load fast with great SEO.
 
-Why choose Astro over another web framework? Here are five core design principles to help explain why we built Astro, the problems that it exists to solve, and why Astro may be the best choice for your project or team.
+:::tip
+Explore the [collection of Astro themes](https://astro.build/themes/) for blogs, portfolios, docs, landing pages, SaaS, marketing, ecommerce sites, and more!
+::: 
```

Comment:
The position of this tip could be better before `Developer-focused` heading from my perspective

Reply:
Ah, so my thinking was, it comes after where we say "content-driven sites" in case people are like, "what does that mean??" The tip here makes it super clear what kind of websites we're talking about.

Given that context, how do you feel about the tip here?

Reply:
True, I still google `content-driven` every now and then, I believe its good position now.
I could also see the tip being well placed after the `Content Driven` Paragraph and right before `Server-first` heading

Reply:
Thanks, then I think the beginning for immediate context makes sense, and then we're not splitting up the design principles. :+1: